druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -2174,6 +2174,14 @@ public RelNode apply(RelBuilder b) {
         .queryContains(druidChecker("'queryType':'timeseries'"));
   }
 
+  /** Test to make sure the "not" filter form is different than "or" & "and"
+   * */
+  @Test public void testNotFilterForm() {
+    String sql = "select count(distinct \"the_month\") from "
+            + "\"foodmart\" where \"the_month\" <> \'October\'";
+    sql(sql, FOODMART)
```

Comment:
can you add the expected druid filter and/or plan to make sure that druid is actually doing the filter and not calcite ?